statistics/cmsketch.go

```diff
@@ -40,15 +42,14 @@ type CMSketch struct {
 	count        uint64 // TopN is not counted in count
 	defaultValue uint64 // In sampled data, if cmsketch returns a small value (less than avg value / 2), then this will returned.
 	table        [][]uint32
-	topN         map[uint64][]topNMeta
+	topN         map[uint64][]TopNMeta
 }
 
-// topNMeta is a simple counter used by BuildTopN
-type topNMeta struct {
-	h1    uint64
+// TopNMeta is a simple counter used by BuildTopN.
+type TopNMeta struct {
 	h2    uint64
```

Comment:
Since `h1` is removed. We can just rename `h2` to `h`?

Reply:
I think it is better not to because we always use `h1` and `h2` together.

Reply:
Yeah, i read the other codes know the meaning of its name. So we'd better add some comment.